phpBB/phpbb/routing/router.php

```diff
@@ -172,8 +173,15 @@ public function get_routes()
 	{
 		if ($this->route_collection == null || empty($this->routing_files))
 		{
-			$this->find_routing_files($this->extension_manager->all_enabled(false))
-				->find($this->phpbb_root_path);
+			if ($this->extension_manager !== null)
+			{
+				$this->find_routing_files($this->extension_manager->all_enabled(false))
+					->find($this->phpbb_root_path);
+			}
+			else
+			{
+				$this->find($this->phpbb_root_path);
```

Comment:
you still have to call `find_routing_files()` with an empty array to fill the `$routing_files` array with the core one.